### PR TITLE
Issue #590: updated and reworded integration test section of the CRUD mission

### LINKED ISSUE #590: CRUD integration tests remove application
In section 2.3.3, it might be worth documenting that the integration tests for the Vert.x CRUD booster completely remove the application when the tests are complete.  If you navigate to the application overview page after the tests have been run, you get the "Add to Project" button (i.e. an empty project).

### PR DESCRIPTION
**Pull request by inoxx03**
fixes #590 

@rdruss @Ladicek WDYT?

### CODE REVIEW

File: docs/topics/crud-mission-integration-testing.adoc [hunk 1]
```diff
@@ -1,8 +1,22 @@
 [[crud-integration-testing]]
 = Running Integration Tests
 
-This booster contains a set of integration tests.
-To run them, you must be connected to an OpenShift instance and select the project that contains your CRUD application and database.
+This booster includes a set of integration tests.
+This set of tests is self-contained, meaning that if you run them inside an empty project, they
```
Comment (Ladicek) [inline docs/topics/crud-mission-integration-testing.adoc]: The word "project" can have multiple meanings in this context; I'd suggest saying "inside an empty OpenShift project" explicitly.

Comment (rhoads-zach) [reply docs/topics/crud-mission-integration-testing.adoc]: If you're doing a list mid-sentence, you should probably add a colon after they.

File: docs/topics/crud-mission-integration-testing.adoc [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+
+* deploy a test instance of the application to the project,
+* execute the individual tests on that instance,
+* remove the test instance of the application from the project when the testing is done.
+
+Before running the tests, ensure that you
```
Comment (rhoads-zach) [inline docs/topics/crud-mission-integration-testing.adoc]: same as above

**Comment by inoxx03** [comment]
@rhoads-zach @Ladicek Thank you for the feedback:) I amended the wording a little. Please let me know what you think.

**Comment by rhoads-zach** [comment]
LGTM, merging.

**Pull request merged**